index.html

```diff
@@ -9,4 +10,355 @@
+
+</head>
+
+<body>
+
+    <!--Navbar-->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top py-1">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#"><img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/f.png"
+                    class="m1" width="95" height="50"></a>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <input class="form-control" type="search" placeholder="Search for Products,brand and more"
+                    aria-label="Search">
+                <button class="btn"> <i class="fa fa-search"></i></button>
+                <button class="b1">Login</button>
+                <a class="nav-link dropdown-toggle"> More </a>
+
+                <button class="cart"><i class="fa fa-shopping-cart cart"> Cart </i></button>
+            </div>
+        </div>
+    </nav>
+
+    <!--Navbar End-->
+
+
+    <!-- Deal section-->
+    <div class="Container-main pt-lg-5">
+        <section>
+            <div class="container p-0 pt-lg-5 ml-10">
+                <div class="row">
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (8).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Top offers </h7>
+                        </div>
+                    </div>
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (7).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Grocery </h7>
+                        </div>
+                    </div>
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image.png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Mobiles </h7>
+                        </div>
+                    </div>
+
+
+                    <div class="col">
+                        <div class="well">
+                            <div class="overlay">
+                                <img src="/home/akshaypatel/Dekstop/Simform/HTML & CSS/simple/MicrosoftTeams-image (2).png"
+                                    width="60%" height="60%">
+                            </div>
+                            <h7> Fashion </h7>
+                            <select class="value">
```

Comment:
@Akshay8122 instead of an empty select tag, use SVG icons to show arrow in all places where applicable